src/features/courses/course-route.ts

```diff
@@ -275,6 +297,72 @@ router.put('/topic/:id',
         }
     }));
 
+router.get('/assessment/topic/:id/start',
+    authenticationMiddleware,
+    validate(createAssessmentVersionValidation),
+    // This is due to a typescript issue where the type mismatches extractMap
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    asyncHandler(async (req: RederlyExpressRequest<any, unknown, CreateAssessmentVersionRequest.body, CreateAssessmentVersionRequest.query>, _res: Response, next: NextFunction) => {
+        const params = req.params as CreateAssessmentVersionRequest.params;
+        if (_.isNil(req.session)) {
+            throw new Error(Constants.ErrorMessage.NIL_SESSION_MESSAGE);
+        }
+        const user = await req.session.getUser();
+
+        let topic = await courseController.getTopicById({id: params.id}); // includes TopicOverrides
+        const topicInfo = await courseController.getTopicAssessmentInfoByTopicId({ topicId: topic.id, userId: user.id }); // ordered by startDate, includes overrides
```

Comment:
I know @PanopticaRising like keeping these without spacing, however I personally do not like this

The pro is brevity keeping the code looking shorted

The con IMO is that it is less readable (easier to read a list with newlines) and more likely to have merge conflicts